### DISCUSSION TITLE
GPU passthrough with WSL2g

### DISCUSSION THREAD

**Discussion by evilz** [sickcodes/Docker-OSX#428] [Q&A]
It is possible to get OpenGl running in macos under Wsl2g ?

**Comment by sickcodes** [comment]
Depends, if you have a support card in the OpenCore list: https://dortania.github.io/OpenCore-Install-Guide/macos-limits.html#gpu-support

**Reply by evilz** [reply]
😭Got a RTX 2080, so if I'm right it's a Turing architecture that is not supported yet.
